Makefile

```diff
@@ -1,209 +1,18 @@
-# Old-skool build tools.
-#
-# Targets (see each target for more information):
-#   all: Build code.
-#   build: Build code.
-#   check: Run verify, build, unit tests and cmd tests.
-#   test: Run all tests.
-#   run: Run all-in-one server
-#   clean: Clean up.
+all: build
+.PHONY: all
 
-OUT_DIR = _output
-OS_OUTPUT_GOPATH ?= 1
+build:
+	go build -o _output/bin/dockerregistry github.com/openshift/image-registry/cmd/dockerregistry
```

Comment:
Unclear what  scope this is in but I think we will need:

 - ability to cross-build
 - ability to add `ldflag`s
 - ability to add build tags

Reply:
should not need to cross-build, the registry binary runs in a linux image, it doesn't run on bare metal.

Reply:
@smarterclayton still maintains we need to be able to ship the binary itself